pkg/controllers/vdb/dbremovenode_reconciler.go

```diff
@@ -146,6 +148,14 @@ func (d *DBRemoveNodeReconciler) removeNodesInSubcluster(ctx context.Context, sc
 			return ctrl.Result{}, fmt.Errorf("failed to update subcluster status: %w", err)
 		}
 
+		if vmeta.UseVProxy(d.Vdb.Annotations) {
```

Comment:
Move the condition "sc.Size == 0" here since we don't want to build proxy deployment when sc.Size != 0 and it will be more readable.

Reply:
moved in https://github.com/vertica/vertica-kubernetes/pull/980/commits/6ac1b322c3ca3e79772bf448f5d47c8a63fa304d